### PR TITLE
bug(fab): fix alignment issues

### PR DESCRIPTION
**Pull request by amcdnl**
This PR:

- Fixes alignment of icons + buttons in fabs
- Decreases font size of text in mini-fabs

img

### CODE REVIEW

File: src/lib/button/button.scss [hunk 1]
```diff
@@ -24,11 +24,12 @@
...
 
 .mat-fab {
-  @include mat-fab($mat-fab-size, $mat-fab-padding);
+  @include mat-fab($mat-fab-size);
 }
 
 .mat-mini-fab {
-  @include mat-fab($mat-mini-fab-size, $mat-mini-fab-padding);
+  @include mat-fab($mat-mini-fab-size);
+  font-size: 80%;
```
Comment (jelbourn) [inline src/lib/button/button.scss]: I don't see anything in the spec that talks about font-size for the mini fab- what are you basing this on?

Comment (amcdnl) [reply src/lib/button/button.scss]: Basing on it was unproportionate in the smaller use case. Happy to remove if you would like.

File: src/lib/button/_button-base.scss [hunk 2]
```diff
@@ -96,7 +94,6 @@ $mat-mini-fab-padding: 8px !default;
   border-radius: $mat-fab-border-radius;
   width: $size;
   height: $size;
-  padding: 0;
```
Comment (tinayuangao) [inline src/lib/button/_button-base.scss]: Not sure about this change. On my demo the text in the fab/mini-fab are off. 
image

File: src/lib/button/button.scss (hunk 1 above)
Comment (amcdnl) [reply src/lib/button/button.scss]: Removed.

File: src/lib/button/_button-base.scss (hunk 2 above)
Comment (amcdnl) [reply src/lib/button/_button-base.scss]: Must have been cache, fixed.

**Comment by amcdnl** [comment]
@josephperrott - any feedback or can we merge?

**Comment by josephperrott** [comment]
@amcdnl go ahead and rebase and we can try to get it in.

**Comment by jelbourn** [comment]
Closing this since we're revamping button to be based on MDC web which should fix this separately

**Pull request closed without merging**